packages/vm/package.json

```diff
@@ -101,16 +101,5 @@
   "bugs": {
     "url": "https://github.com/ethereumjs/ethereumjs-vm/issues"
   },
-  "homepage": "https://github.com/ethereumjs/ethereumjs-vm",
-  "nyc": {
```

Comment:
Removing the `nyc` key, I don't think this is needed, if it is we should add it to `.nycrc` instead.